sdk/core/src/error/http_error.rs

```diff
@@ -1,4 +1,4 @@
-use crate::{headers, Response};
+use crate::{headers, response, Response, INVALID_BODY};
```

Comment:
nit: I'm not sure exposing `INVALID_BODY` from the top of the crate is the right thing to do. It's used rarely enough that keeping it in a submodule would make sense IMO.